docs/docs/05-reusable-components.md

```diff
@@ -227,7 +227,7 @@ Counter.defaultProps = { initialCount: 0 };
 
 ### No Autobinding
 
-Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or arrow functions.
+Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or arrow `=>` functions.
```

Comment:
I actually think this is pretty extraneous and awkward but wouldn't be opposed to linking to some definitive source on arrow functions.